packages/widget_toolkit_pin/README.md

```diff
@@ -0,0 +1,270 @@
+```
+
+**Note:** The `WidgetToolkitTheme` comes from the **Widget Toolkit** package which already presence in the **Widget Toolkit Pin** package.
+It can be imported with the following line:
+```dart
+import 'package:widget_toolkit/widget_toolkit.dart';
+```  
+
+Step 3: Create an implementation of `BiometricsLocalDataSource`, for example:
+```dart
+class ProfileLocalDataSource implements BiometricsLocalDataSource { 
+	static const _areBiometricsEnabled = 'areBiometricsEnabled';
+	
+	Future<SharedPreferences> get _storageInstance => SharedPreferences.getInstance();
+	
+	@override  
+	Future<bool> areBiometricsEnabled() async {  
+	  final storage = await _storageInstance;  
+	  return storage.getBool(_areBiometricsEnabled) ?? false;  
+	}
+	
+	@override  
+	Future<void> setBiometricsEnabled(bool enable) async {  
+	  final storage = await _storageInstance;  
+	  await storage.setBool(_areBiometricsEnabled, enable);  
+	}
+}
+```
+
+Step 6: Create an implementation of `PinCodeService`. In this example we use double encryption. The 
+pin code is first encrypted on application level and then the encrypted value is again encrypted on
+operating system level, by using the `FlutterSecureStorage` instance. In your implementation, you 
+are free to choose the types of encryption. In the example two other packages are used:
+[encrypt](https://pub.dev/packages/encrypt) and [flutter_secure_storage](https://pub.dev/packages/flutter_secure_storage)
+In order for the `flutter_secure_storage` plugin to work on your desired platforms, follow the 
+integration instructions.
+```dart
+class AppPinCodeService implements PinCodeService {
+  static const _isPinCodeInStorage = 'pinCode';
+
+  FlutterSecureStorage get flutterSecureStorage => const FlutterSecureStorage();
+
+  @override
+  Future<bool> isPinCodeInSecureStorage() async {
+    var isPinCodeInSecureStorage =
+    await flutterSecureStorage.read(key: _isPinCodeInStorage);
+
+    if (isPinCodeInSecureStorage == null) {
+      return Future.value(false);
+    }
+    return Future.value(true);
+  }
+
+  @override
+  Future<String> encryptPinCode(String pinCode) async {
+    // App specific encryption
+    final key = Key.fromSecureRandom(32);
+    final iv = encrypt.IV.fromSecureRandom(16);
+    final encrypter = Encrypter(AES(key));
+    final encryptedString = encrypter.encrypt(pinCode, iv: iv).base64;
+
+    // Platform specific encryption, save the pin in secure storage
+    await flutterSecureStorage.write(
+        key: _isPinCodeInStorage, value: encryptedString);
+    return Future.value(encryptedString);
+  }
+
+  @override
+  Future<int> getPinLength() => Future.value(3);
+
+  @override
+  Future<bool> verifyPinCode(String pinCode) async {
+    var pinFromStorage =
+    await flutterSecureStorage.read(key: _isPinCodeInStorage);
+
+    if (pinCode == pinFromStorage) {
+      return Future.value(true);
+    }
+    return Future.value(false);
```

Comment:
The whole code block:
```dart
if (pinCode == pinFromStorage) {
    return Future.value(true);
}
return Future.value(false);
```

can be replaced with a single line:

```dart
return pinCode == pinFromStorage;
```

This should work without the `Future.value(a_value)`, since the function is marked as `async`.

Reply:
I have changed the code section to `return pinCode == pinFromStorage;`